.github/workflows/build.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup Toolchain
-        run: ./scripts/setup-debian.sh
+        run: sudo ./scripts/setup-debian.sh ${{ matrix.build.variant }}
```

Comment:
Why？

Reply:
the script internal use `sudo apt-get`, this is move `sudo` to the script external.

Reply:
AFAIK, `sudo` is useless in  GitHub Actions, it's enough to just run `apt-get`.

Reply:
OK, i try it

Reply:
![CleanShot 2024-06-21 at 13 49 22@2x](https://github.com/estkme-group/lpac/assets/3842474/decae0ea-696d-4305-9f32-6fe59e2741fd)